content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -24,20 +24,24 @@ icon:bullhorn[size=2x]{nbsp} To display custom dashboards, you must set the _app
 
 == Prometheus Configuration
 
-Kiali runtimes monitoring feature works exclusively with Prometheus, so it must be configured correctly to pull your application metrics.
+Kiali custom dashboards work exclusively with Prometheus, so it must be configured correctly to pull your application metrics.
 
 If you are using the default Istio installation, your Prometheus instance should already be configured as shown below and you can skip to the next section.
 
 But if you want to use another Prometheus instance, please refer to link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config[Prometheus documentation] to setup `kubernetes_sd_config` for pods. As a reference, link:https://github.com/istio/istio/blob/907aa731c3f76ad21faac98614751e8ab3531893/install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml#L229[here is] how it is configured in Istio.
 
-It is important to preserve label mapping, so that Kiali can filter on _app_ and _version_, which is done with this configuration:
+It is important to preserve label mapping, so that Kiali can filter on _app_ and _version_, and to have the _namespace_ label. See the `relabel_configs` that would allow this:
 
 ```yaml
+      relabel_configs:
       - action: labelmap
         regex: __meta_kubernetes_pod_label_(.+)
+      - source_labels: [__meta_kubernetes_namespace]
+        action: replace
+        target_label: namespace
 ```
 
-Additionally, you must configure the Prometheus server URL in the _Kiali CR_:
+Additionally, you must configure the Prometheus server URL in the _Kiali CR_ or _ConfigMap_:
```

Comment:
Should we mention that it's the CR when using the operator and otherwise the config map?

Reply:
+1